.gitmodules

```diff
@@ -1,6 +1,9 @@
 [submodule "src/components/lib/ps/ps"]
 	path = src/components/lib/ps/ps
-	url = https://github.com/gwsystems/ps.git
```

Comment:
I think we could first update the ps port, so we don't need to change the url here